packages/chrome-extension/README.md

```diff
@@ -154,6 +154,8 @@ curl  -X PATCH https://api.clerk.com/v1/instance \
       -d '{"allowed_origins": ["chrome-extension://extension_id_goes_here"]}'
 ```
 
+Setting the allowed_origins is **REQUIRED** for both **Development** and **Production** instances when using the WebSSO use case.
```

Comment:
@dimkl Let's make this a separate section to make it more clear.

Reply:
I moved it to a top level section called `Deploy to production`. PTAL